### PR TITLE
Expand row

### PR DESCRIPTION
**Pull request by jgiardino**
## Description

Compared updates I made with updates June made, and merged our conflicts.

Related PR: https://github.com/patternfly/patternfly/pull/487/files
Related story: https://patternfly.atlassian.net/browse/PTNFLY-452
## Changes
- For specific details about CSS changes, refer to the callouts in the images below.
- Made updates requested in https://github.com/patternfly/patternfly/pull/487/files
- I investigated including the blue border below the selected node in compound expansion, but this requires modifying how padding is applied to the elements. This also results in a layout that's too tall in the case where the nodes wrap due to padding being applied to the individual nodes. It might be possible to implement this a different way, but this would require further investigation.
- I also did not modify the colors applied to the compound nodes when active and on hover, since I'm not sure if we want to support the case where icons are not included (i.e. if only icons have the color change and they aren't included, then nothing in the node has a color change)
## Link to rawgit and/or image

Examples Before and After Updates
1. I modified the border styles so that left and right blue borders are applied to the expanded state.
2. I also modified the border styles to avoid the double border that occurs when one list row is expanded, and the next list row is expanded or selected.
3. I replaced the <ul> implementation with <dl class="dl-horizontal">

listviewcssupdates-before-callouts

listviewcssupdates-after-callouts
## PR checklist (if relevant)
- [ ] **Cross browser**: works in IE9
- [ ] **Cross browser**: works in IE10
- [ ] **Cross browser**: works in IE11
- [ ] **Cross browser**: works in Edge
- [ ] **Cross browser**: works in Chrome
- [ ] **Cross browser**: works in Firefox
- [ ] **Cross browser**: works in Safari
- [ ] **Cross browser**: works in Opera
- [ ] **Responsive**: works in extra small, small, medium and large view ports.
- [ ] **Preview the PR**: An image or a URL for designer to preview this PR is provided.

### CODE REVIEW

**Comment by andresgalante** [comment]
This PR is a continuation of #487

**Comment by andresgalante** [comment]
Rawgit links are:
https://rawgit.com/jgiardino/patternfly/expandRow-dist/dist/tests/list-view-compound-expansion.html
and
https://rawgit.com/jgiardino/patternfly/expandRow-dist/dist/tests/list-view-simple-expansion.html#

File: src/less/list-view.less [hunk 1]
```diff
@@ -230,5 +251,62 @@
...
+  background: #fff;
+  border-top: solid 1px @list-group-active-border;
+  box-sizing: content-box;
+  margin: -1px -15px 0;
+  order: 3;
+  padding: 10px 15px;
+  position: relative;
+  width: 100%;
+}
+.list-view-pf-expand-close {
```
Comment (junezhang) [inline src/less/list-view.less:298]: We can remove ".list-view-pf-expand-close" ".list-view-pf-expand-chart"
In the html page, using the below code snippet based on Andres's comment.

```
<div class="close">
    <span class="pficon pficon-close"></span>
</div>
```

**Comment by junezhang** [comment]
@jgiardino Thanks for your update. Your code is helpful for me. and all is okay for me. just one minor point I left comment in the css code.

**Review by junezhang — APPROVED** [review]

File: src/less/list-view.less (hunk 1 above)
Comment (andresgalante) [reply src/less/list-view.less:298]: 👍

**Comment by LHinson** [comment]
@kybaker do you think we need to include additional stories to figure out how to get that blue highlight detail underneath or is the design ok as implemented? 

@jgiardino I've compared the implementation with the pattern page and it matches (minus what you've already mentioned). Therefore LGTM. 

Looping in @dgutride since he will be running with this. Also would be good to have Matt C. check it since he was the designer on this. I'll loop in Matt... can't find him on GitHub atm.

File: src/less/list-view.less (hunk 1 above)
Comment (jgiardino) [reply src/less/list-view.less:298]: Good catch! Thanks for reviewing this!

**Comment by jgiardino** [comment]
Closing this one, Created #507 instead

**Pull request closed without merging**